src/app/pages/wallets/wallets.ts

```diff
@@ -748,7 +748,7 @@ export class WalletsPage {
 
   public checkCardExistListPrimary(wallet, token?) {
     let data = JSON.parse(localStorage.getItem("listHome"));
-    let isExist = _.find(data, item => item.walletId === wallet.id && item?.tokenId === token?.tokenId);
+    let isExist = _.find(data, item => item.walletId === wallet.id && item.tokenId === token?.tokenId);
     return !!isExist;
```

Comment:
Check case of isExist = 0 ( first element of data found )
!!isExist will return false 
=> Suggest : check isExist > -1